test/libsolidity/SemanticTest.cpp

```diff
@@ -52,6 +53,7 @@ SemanticTest::SemanticTest(string const& _filename, langutil::EVMVersion _evmVer
 	m_lineOffset(m_reader.lineNumber()),
 	m_enforceViaYul(enforceViaYul)
 {
+	m_builtins["storage_empty"] = bind(&SemanticTest::builtinStorageIsEmpty, this, std::placeholders::_1);
```

Comment:
Maybe it's better to create a function `initializeBuiltins` and use lambda functions inside that instead of `bind`?